go-sdk/bundle/bundlev1/bundlev1server/doc.go

```diff
@@ -0,0 +1,25 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Doesn't have to be done but bundleserver can live in `pkg/bundles`? Or is there a reason for new top level directory for server and client for bundles?

Reply:
pkg/bundles/bundlev1/server/     <= server implementation  
pkg/bundles/bundlev1/client/     <= client implementation


Maybe

Reply:
I didn't make this clear I guess, but the intent is:

`bundles/` <-- things users will use in writing their tasks/bundles
`pkg/` <-- anything "quasi-internal" i.e. I don't expect or want users to use it directly, but I also don't believe in putting things truly private (which `internal/` is)

Reply:
Updated, removed the `pkg/bundles/client` folder